### ISSUE TITLE
Calabash-iOS Failed to install the app on iOS 11, XCode 9.1, Sierra 10.12.6

### ISSUE THREAD

**Issue by nhquijano** [calabash/calabash-ios#1346]
Hi! 

I have been encountering this error when running on iOS 11 using XCode 9.1
```
An error was encountered processing the command (domain=NSPOSIXErrorDomain, code=2):
  Failed to install the requested application
  An application bundle was not found at the provided path.
  Provide a valid path to the desired application bundle.
  
  This usually means your CoreSimulator processes need to be restarted.
  
  You can restart the CoreSimulator processes with this command:
  
  $ bundle exec run-loop simctl manage-processes
  
   (RuntimeError)
  ./features/support/01_launch.rb:18:in `launch'
  ./features/support/01_launch.rb:232:in `Before'
```

This is happening 100% of the time. Tried running the suggested solution in terminal but nothing happens.

**Comment by jmoody** [comment]
The binary that we use to interact with physical devices is broken for Xcode >= 9.0.1

Expect a fix shortly.

Until then, use Xcode 9.0.

**Comment by jmoody** [comment]
Fixed in run-loop 2.6.2

Thanks for reporting.

Thanks for your patience.

**Comment by hariDasu** [comment]
wont install in ios11 and arm64 will not build
